sycl/source/detail/global_handler.cpp

```diff
@@ -71,6 +71,49 @@ std::atomic_uint ObjectUsageCounter::MCounter{0};
 GlobalHandler::GlobalHandler() = default;
 GlobalHandler::~GlobalHandler() = default;
 
+void GlobalHandler::InitXPTIStuff() {
```

Comment:
Nit; Although I love the name, I think it would better to omit the `Stuff` part. In my opinion it doesn't add anything to the readability, so better keep it short.

Reply:
done